src/call_frame.rs

```diff
@@ -19,28 +19,62 @@ pub struct CallFrame {
     pub aux_heap: Heap,
     pub code_page: Vec<U256>,
     pub pc: u64,
-    /// Storage for the frame using a type that implements the Storage trait.
-    /// The supported types are InMemory and RocksDB storage.
+    // TODO: Storage is more complicated than this. We probably want to abstract it into a trait
+    // to support in-memory vs on-disk storage, etc.
```

Comment:
I think this is a wrong change after merge, right?

Reply:
Yes, probably some copy paste there because of the change of directory or something